File: src/main/java/pr/eleks/we_at_her/dto/WeatherSampleDto.java
package pr.eleks.we_at_her.dto;


public class WeatherSampleDto extends AbstractWeatherSampleDto {
    public WeatherSampleDto() {
    }

    public WeatherSampleDto(String cityName, float temperature, float feelsLike, float pressure, int humidity, int clouds, int cityId, int time, float latitude, float longitude) {
        this.cityName = cityName;
        this.temperature = temperature;
        this.feelsLike = feelsLike;
        this.pressure = pressure;
        this.humidity = humidity;
        this.clouds = clouds;
        this.cityId = cityId;
        this.time = time;
        this.latitude = latitude;
        this.longitude = longitude;
    }
}

File: src/main/kotlin/pr/eleks/we_at_her/dto/weather/WeatherSampleDto.kt
package pr.eleks.we_at_her.dto.weather;


data class WeatherSampleDto(
        override var id: Long? = null,
        override var cityName: String? = null,
        override var temperature: Float? = null,
        override var feelsLike: Float? = null,
        override var pressure: Float? = null,
        override var humidity: Int? = null,
        override var clouds: Int? = null,
        override var cityId: Int? = null,
        override var time: Int? = null,
        override var latitude: Double = 0.0,
        override var longitude: Double = 0.0
) : AbstractWeatherSampleDto() {
    constructor(cityName: String?, temperature: Float?, feelsLike: Float?, pressure: Float?, humidity: Int?,
                clouds: Int?, cityId: Int?, time: Int?, latitude: Double, longitude: Double
    ) : this(null, cityName, temperature, feelsLike, pressure, humidity,
            clouds, cityId, time, latitude, longitude
    )
}